src/OgRoleInterface.php

```diff
@@ -5,11 +5,12 @@
  * Contains Drupal\og\OgRoleInterface.
  */
 namespace Drupal\og;
+use Drupal\user\RoleInterface;
 
 /**
  * Provides an interface defining an OG user role entity.
  */
-interface OgRoleInterface {
+interface OgRoleInterface extends RoleInterface {
```

Comment:
I know what this is, PHP5.x handles interface inheritance differently. On PHP5 you cannot extend a class and implement an interface that both extend a class that contains certain constants. It will try to define the constants twice. In PHP7 this is fixed.

So in practice:
1. `RoleInterface` defines a number of constants.
1. `OgRole` implements `OgRoleInterface` which implements `RoleInterface`. `OgRole` now inherits the constant from `RoleInterface`.
1. `OgRole` also extends `Role` which in turn also implements `RoleInterface`. Now the fatal error is thrown since PHP5 will try to define the constants again.

So if we want to retain compatibility with PHP5 we cannot let `OgRoleInterface` extend `RoleInterface`.